README.md

```diff
@@ -53,22 +55,17 @@ by supporting different modes.
 
 ## Requirements
 
-autohooks uses `/usr/bin/env` with the `-S` argument to support
-[pipenv](#pythonpath-mode) and [poetry](#poetry-mode) modes. The `-S` argument
-got introduced with coreutils [8.30 (2018-07-01)](https://github.com/coreutils/coreutils/blob/master/NEWS#L241).
-This version is available in Debian Buster and since Ubuntu 19.04.
-
-If you are using an older distribution not providing coreutils 8.30 or newer please choose the [python path mode](#pythonpath-mode).
-
 autohooks supports Python 3.5+. For development Python 3.6+ is required.
 
 ## Modes
 
-Currently three modes for using autohooks are supported:
+Currently five modes for using autohooks are supported:
 
 * `pythonpath`
 * `pipenv`
+* `pipenv_mutiline`
 * `poetry`
+* `poetry_mutiline`
```

Comment:
```suggestion
* `poetry_multiline`
```

Reply:
I think I may have done the update wrong, but this was removed as an option for the user to configure.